modules/core/src/test/scala/org/scalasteward/core/forge/ForgeTypeTest.scala

```diff
@@ -0,0 +1,58 @@
+package org.scalasteward.core.forge
+
+import munit.FunSuite
+import org.scalasteward.core.TestSyntax._
+import org.scalasteward.core.data.{Repo, Update}
+import org.scalasteward.core.forge.ForgeType.{GitHub, GitLab}
+import org.scalasteward.core.git
+
+class ForgeTypeTest extends FunSuite {
```

Comment:
Now that the implementation code is in `ForgeType`, this is mainly just a rename of `ForgePackageTest` to `ForgeTypeTest` - and correction to all assertions so that thy assert the correct `head` values).